msteams-platform/tabs/how-to/authentication/auth-aad-sso.md

```diff
@@ -8,7 +8,7 @@ keywords: teams authentication SSO Microsoft Azure Active Directory (Azure AD) s
 
 # Single sign-on (SSO) support for tabs
 
-Users sign in to Microsoft Teams through their work, school, or Microsoft account that is Office 365, Outlook, you can take the advantage by allowing a single sign on to authorize your Teams tab or task module on desktop or mobile clients. If a user sign in once, they don't have to sign in again on another device as they're signed in automatically. Also, your access token is prefetched to improve performance and load times.
+Users sign in to Microsoft Teams through their work, school, or Microsoft account that is Office 365, Outlook, you can take the advantage by allowing a single sign on to authorize your Teams tab or task module on desktop or mobile clients. If a user sign in once, they don't have to sign in again on another device as they're signed in automatically. Also, your access token is pre-fetched to improve performance and load times.
```

Comment:
Microsoft account is not Office 365

Reply:
Hello,
Thank you for the comments.
This page content is not used and has been revamped as part of this PR. Please use the following link for review site:  https://review.docs.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/authentication/tab-sso-overview?branch=pr-en-us-5567
Thanks.